src/commands/tick-start.js

```diff
@@ -0,0 +1,52 @@
+import { parser } from 'tickbin-parser'
+import { writeSavedTimer } from './output'
+import db from '../db'
+
+export default { builder, handler : start}
+
+const defTimersDoc = {
+  _id: '_local/timers',
+  timers: []
+}
+
+function builder(yargs) {
+  return yargs
+  .usage('Usage: tick start [message]')
+  .example('tick start', 'start a timer for the current time')
+  .example('tick start "8am"', 'start a timer for 8am')
+  .example('tick start "8am squashing bugs #dev"', 'start a timer for 8am and provide a message for the commit')
+}
+
+function start(argv) {
+  db.get('_local/timers')
+  .then(
+    //  If the timers document exists, pass it to saveTimer. Otherwise pass the
+    //  defTimersDoc
+    timerDoc => saveTimer(timerDoc, argv._[1]),
+    () => saveTimer(defTimersDoc, argv._[1])
```

Comment:
There are cases other than the doc missing where this promise might be rejected. Maybe it's worth checking that `err.name == 'not_found'` to ensure that was the case first? Also, not sure if it was deliberate to use the second parameter to the `then` but you can repeat catches if you want:

```
db.get('_loca/timers')
.then(/* Handle existing timer */)
.catch(/* Handle missing timer, rethrow if it was some other error */)
.then(writeSavedTimer)
.catch(err => console.log('Could not start timer'...))
```